statics/js/components/topology.js

```diff
@@ -692,7 +692,7 @@ var TopologyComponent = {
     },
 
     normalizeMetric: function(metric) {
-      if (metric.Start && metric.Last) {
+      if (metric.Start && metric.Last && !metric.Bandwidth) {
```

Comment:
the test should be (metric.Last - metric.Start) > 0

Reply:
Done.